modules/configuring-haproxy-interval.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/routing-optimization.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-haproxy-interval_{context}"]
+= Configuring HAProxy reload interval
+
+When you update a route or an endpoint associated with a route, OpenShift router updates the configuration for HAProxy. Then HAProxy reloads the updated configuration for those changes to take effect. When HAProxy reloads, it generates a new process that handles new connections using the updated configuration. HAProxy keeps the old process running to handle existing connections until those connections are all closed. When old processes have long-lived connections, these processes can accumulate and consume resources. The default minimum HAProxy reload interval is 5 seconds. You can configure an Ingress Controller using its `spec.tuningOptions.reloadInterval` field to set a longer minimum reload interval.
+
+* Change the minimum HAProxy reload interval of the default Ingress Controller to 15 seconds by running the following command:
++
+[source, terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"tuningOptions":{"reloadInterval":"15s"}}}'
+----
+
+[WARNING]
```

Comment:
Consider putting this above the procedure

Reply:
Addressed